### DISCUSSION TITLE
Hello World

### DISCUSSION THREAD

**Discussion by whyman** [gerbera/gerbera#1055] [General]
## 👋 Hello World!
  We’re trailing Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about how you use Gerbera!.

**Comment by KarlStraussberger** [comment]
I started using Mediatomb 0.11 around 2008 or so and was really happy to see its successor Gerbera starting up and getting momentum. Currently, my library consists of almost 30000 music tracks, 2000 video files and 68000 digital images.

Besides from "_playing_" around with gerbera, I'm a software consultant in a small company near Nuremberg, Germany. I wrote my first C++ program in **1992** while in university struggling for a maths degree after reading Bjarne Stroustrup's book from cover to cover. Nowadays, most of my projects are implemented with C#, Angular and bits of JavaScript.
